.github/workflows/crowdin.yaml

```diff
@@ -11,23 +11,26 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
 
-      - name: Set up node
-        uses: actions/setup-node@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: 14
           registry-url: https://registry.npmjs.org
           cache: 'yarn'
 
-      - name: Install dependencies
-        run: yarn install --frozen-lockfile
+      - uses: actions/cache@v3
+        id: install-cache
+        with:
```

Comment:
i dont really follow what's going on here mostly

Reply:
This explicitly puts node_modules in a cache, instead of the global yarn cache. SO that if this yarn.lock has already been installed, we can just copy node_modules over from cache. This works because we run our CI on identical machines, with identical versions of node, so that we can just copy over node_modules across builds. Effectively, this saves about 30s per CI action, because it lets us skip installing node_modules.

Reply:
smart